lib/matplotlib/text.py

```diff
@@ -1410,7 +1410,9 @@ def __call__(self, renderer):
         elif isinstance(self._artist, Transform):
             x, y = self._artist.transform(self._ref_coord)
         else:
-            raise RuntimeError("unknown type")
+            raise RuntimeError(
```

Comment:
This could just use check_isinstance? (Yes, that changes the exception type, but I think the API change is ok, or if you really want to keep the exception type, you could catch the TypeError and  reuse the error message.)